test/Microsoft.AspNet.Razor.Test/Framework/BlockTypes.cs

```diff
@@ -177,27 +177,43 @@ public MarkupBlock(IEnumerable<SyntaxTreeNode> children)
 
     public class MarkupTagHelperBlock : TagHelperBlock
     {
-        public MarkupTagHelperBlock(string tagName)
-            : this(tagName, new Dictionary<string, SyntaxTreeNode>())
+        public MarkupTagHelperBlock(string tagName, bool selfClosing = false)
+            : this(tagName, new Dictionary<string, SyntaxTreeNode>(), selfClosing)
         {
```

Comment:
You might prefer explicitly providing the selfclosing parameter values from each test over this default parameter. I tried that but I found >260 build errors. I don't think it is worth updating that many test data. But, let me know if you have a strong opinion on this.